### DISCUSSION TITLE
How much time does training requires on a macbook, and what are those error messages...?

### DISCUSSION THREAD

**Discussion by mirashindento** [acids-ircam/RAVE#168] [General]
Hi, I am a total noob at model training, I just installed Rave on my MacBook pro, created a folder with 3 small audio files between 3 to 23mb (wav) on it, and executed the preprocess command. 

It is now seemingly stucked at:
`0it [00:00, ?it/s](FileNotFoundError(2, 'No such file or directory'),)`
And there is a data.mbd file created on my output folder announcing 107 GB...

Should I worry? And how much time and memory does this take?

**Comment by caillonantoine** [comment]
This issue has been reported here #160 . Can you check that you have `ffmpeg` installed ?

Training on macbook is not supported at the time ! However, once pytorch fully support the MPS backend I'll work on optimizing training for Mac.

**Reply by mirashindento** [reply]
Thank you for your quick answer Antoine!
ffmpeg wasn't installed, which version should be installed?

Additional question: if not supported on mac, should I go for the google collab you provided:
https://colab.research.google.com/drive/1aK8K186QegnWVMAhfnFRofk_Jf7BBUxl?usp=sharing#scrollTo=yShbolt8EtAK

Also how much time can this generally take? (I want to use my owned model for a project in one week, this will be part of a pre-existing max patch I made for sound installation/performance)

Thanks in advance.

**Reply by caillonantoine** [reply]
You can install ffmpeg using the command [described here](https://github.com/acids-ircam/RAVE#installation) 

Training on colab can be long, it depends on many factors, the dataset you are using, your colab plan, how many times you will be disconnected...

**Comment by ogbabydiesal** [comment]
having a similar issue on my Windows 10 machine with "finding the path" and getting the dreaded "0it [00:00, ?it/s](FileNotFoundError(2, 'No such file or directory'),) error". 

I was actually getting it to start training on my Mac but I wanted to set up this gaming laptop to train because I think it will probably be faster.

I'm using a venv with pycharm. Is there an IDE that might handle dependencies better for a n00b?

I have noticed it does create a 100gb Data file and lock file as well in the outputh path (so it doesn't seem like it's unable to find those paths actually).  It is not creating the yaml file that my Mac created though. 

ffmpeg is installed.
